src/main/scala/com/microsoft/hyperspace/index/sources/iceberg/IcebergRelation.scala

```diff
@@ -123,14 +125,26 @@ class IcebergRelation(spark: SparkSession, override val plan: DataSourceV2Relati
    *
    * File paths should be the same format as "input_file_name()" of the given relation type.
    * For [[IcebergRelation]], each file path should be in this format:
-   *   `/path/to/file`
+   *   `/path/to/file` or `X:/path/to/file`
```

Comment:
`X:` is only for Windows filesystem (e.g., `file:/`), but not applicable for remote storage (e.g., `abfs://`)? If so, could you clarify this a bit in the comment? For example, we can say that we need to take care of a special care if the path URI's scheme is `file://` depending on the OS.

Reply:
Added some comment.